test/road_test.py

```diff
@@ -16,15 +16,27 @@ def test_transition_probs_without_obstacles_are_always_1():
     for a in ACTIONS:
         for next_state, prob, reward in road_test.successors(a):
             assert prob == 1.0
+# -----------------------------------------------------------------------------------------------------
 
-def test_transition_probs_with_one_obstacle_are_1(): #here we test what happens when the car hits/doesn't hit the obstacle
+# The following tests whether we obtain a probability == 1.0 if either a bump or a pedestrian collides or does not collide with the car.
```

Comment:
The test function name and decorator parameters already say this so this comment should be removed. Or if you don't think those are clear enough, you could try making them more descriptive, but I think they're fine.